### PR TITLE
Update viaq-store.yml

### PR DESCRIPTION
**Pull request by sradco**
Removed part that are already part of the ovirt-engine-metrics installation.

@StLuke Please review.

### CODE REVIEW

File: playbooks/viaq-store.yml [hunk 1]
```diff
@@ -113,16 +113,8 @@
         - "openshift-ansible-lookup-plugins"
         - "openshift-ansible-playbooks"
         - "openshift-ansible-roles"
-    - name: Copy inventory and vars.yml
```
Comment (StLuke) [inline playbooks/viaq-store.yml:116]: this wont work as its being installed on fresh CentOS/RHEL and both inv and vars are not present

File: playbooks/viaq-store.yml [hunk 2]
```diff
@@ -155,108 +147,6 @@
...
-        path: /etc/ovirt-engine-metrics/config.yml
-        state: touch
-    - name: Display all variables/facts known for a host
-      debug:
-        msg: "Debug - {{ hostvars[groups['metrics-store'][0]] }}"
-    - name: Save metrics store machine fqdn
-      set_fact:
-        metrics_store_fqdn: "{{ hostvars[groups['metrics-store'][0]]['ansible_fqdn'] }}"
-    - name: add ovirt_env_name parameters to config.yml file
-      lineinfile:
```
Comment (StLuke) [inline playbooks/viaq-store.yml]: I would still like to modify the config.yml in the script, and not manually

File: conf/ansible-inventory [hunk 3]
```diff
@@ -1,32 +0,0 @@
-[OSEv3:children]
```
Comment (StLuke) [inline conf/ansible-inventory:1]: do we already store these inside oVirt?

File: playbooks/viaq-store.yml [hunk 4]
```diff
@@ -113,16 +113,8 @@
...
-    - name: Copy inventory and vars.yml
-      copy:
-        src: "../conf/{{ item }}"
-        dest: /root/
-        mode: 0644
-      with_items:
-        - "vars.yml"
-        - "ansible-inventory-openshift"
     - name: Run viaq playbooks
-      shell: ANSIBLE_LOG_PATH=/tmp/ansible.log ansible-playbook -v -e @/root/vars.yml -i /root/ansible-inventory-openshift /usr/share/ansible/openshift-ansible/playbooks/byo/config.yml
```
Comment (StLuke) [inline playbooks/viaq-store.yml:125]: I need to make this a variable to deploy based on parameter either origin or openshift

**Review by StLuke — CHANGES_REQUESTED** [review]
I'll try to test this in early 4.2.3, looks good, apart from few commends that would block my automation. I can alter them while testing.

File: playbooks/viaq-store.yml (hunk 1 above)
Comment (sradco) [reply playbooks/viaq-store.yml:116]: I generate and copy the inventory and vars.yml with the namespaces. So the files should already be on the metrics store machine.

File: playbooks/viaq-store.yml (hunk 4 above)
Comment (sradco) [reply playbooks/viaq-store.yml:125]: Updated to a variable with a default for upstream path.

File: conf/ansible-inventory (hunk 3 above)
Comment (sradco) [reply conf/ansible-inventory:1]: yes. I copy it to the metrics store machine.

File: playbooks/viaq-store.yml (hunk 2 above)
Comment (sradco) [reply playbooks/viaq-store.yml]: Added back the relevant tasks.

**Review by StLuke — APPROVED** [review]

**Pull request merged**